dev-python/cython/cython_python-0.27.3.recipe

```diff
@@ -3,11 +3,13 @@ DESCRIPTION="Cython is an optimising static compiler for both the Python \
 programming language and the extended Cython programming language (based on \
 Pyrex). It makes writing C extensions for Python as easy as Python itself."
 HOMEPAGE="http://cython.org/"
-COPYRIGHT="2007-2016 Stefan Behnel, Robert Bradshaw, et al."
+COPYRIGHT="2007-2017 Stefan Behnel, Robert Bradshaw, et al."
 LICENSE="Apache v2"
-REVISION="2"
-SOURCE_URI="https://pypi.python.org/packages/b7/67/7e2a817f9e9c773ee3995c1e15204f5d01c8da71882016cac10342ef031b/Cython-0.25.2.tar.gz"
-CHECKSUM_SHA256="f141d1f9c27a07b5a93f7dc5339472067e2d7140d1c5a9e20112a5665ca60306"
+REVISION="1"
+REPLACES="python_cython-0.25.2"
```

Comment:
This REPLACES is useless, right?

Reply:
The recipe is renamed from python_cython to cython_python.

The REPLACES is needed to make sure pkgman does the update in that case.